src/main/java/software/amazon/smithy/lsp/ext/SmithyProject.java

```diff
@@ -232,4 +359,24 @@ private static List<File> downloadExternalDependencies(SmithyBuildExtensions ext
     private static List<File> onlyExistingFiles(Collection<File> files) {
         return files.stream().filter(File::isFile).collect(Collectors.toList());
     }
+
+    // TODO: Remove this Class once the change in https://github.com/awslabs/smithy/pull/1192 is available.
```

Comment:
This is available now

Reply:
This got merged after 1.21.0. It can get picked up once 1.22.0 is released.